contracts/utils/contracts/src/MixinAuthorizable.sol

```diff
@@ -0,0 +1,121 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/Ownable.sol";
+import "./interfaces/IAuthorizable.sol";
+
+
+contract MixinAuthorizable is
+    Ownable,
+    IAuthorizable
+{
+    /// @dev Only authorized addresses can invoke functions with this modifier.
+    modifier onlyAuthorized {
+        require(
+            authorized[msg.sender],
+            "SENDER_NOT_AUTHORIZED"
```

Comment:
As discussed offline, let's add rich reverts to this contract and move the old `asset-proxy` versions into an `archived` directory in the `asset-proxy` package. The already deployed proxies should inherit the archived versions of this and `Ownable`.